planner/core/find_best_task.go

```diff
@@ -112,6 +112,10 @@ func (p *baseLogicalPlan) findBestTask(prop *property.PhysicalProperty) (bestTas
 		// Next, get the bestTask with enforced prop
 		prop.Cols = []*expression.Column{}
 	}
+	//
```

Comment:
Please remove this line, you can reference existing comment format in the code base.